content/docs/user-guide/managing-external-data.md

```diff
@@ -1,83 +1,94 @@
 # Managing External Data
 
 There are cases when data is so large, or its processing is organized in a way
-that you would like to avoid moving it out of its external/remote location. For
-example from a network attached storage (NAS) drive, processing data on HDFS,
-running [Dask](https://dask.org/) via SSH, or having a script that streams data
-from S3 to process it. External outputs and
-[external dependencies](/doc/user-guide/external-dependencies) provide a way for
-DVC to control data outside of the <abbr>project</abbr> directory.
+such that its preferable to avoid moving it from its external/remote location.
+For example data on a network attached storage (NAS) drive, processing data on
```

Comment:
remove `drive` in the `(NAS) drive`

Reply:
Removed "drive" from both docs in 23be1528.